test/parallel/test-http2-util-headers-list.js

```diff
@@ -1,14 +1,14 @@
 // Flags: --expose-internals
 'use strict';
 
-// Tests the internal utility function that is used to prepare headers
-// to pass to the internal binding layer.
+// Tests the internal utility functions that are used to prepare headers
+// to pass to the internal binding layer and to build a header object
```

Comment:
Could you add a dot at the end of this sentence, please?

Reply:
Sure, will do that soon. As #21296 landed I have to update doc for set-cookie also.

Reply:
Done